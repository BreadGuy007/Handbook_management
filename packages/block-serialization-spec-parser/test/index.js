/**
 * Internal dependencies
 */
import { parse } from '../';

describe( 'block-serialization-spec-parser', () => {
	test( 'parse() works properly', () => {
		const result = parse(
			'<!-- wp:core/more --><!--more--><!-- /wp:core/more -->'
		);

		expect( result ).toMatchSnapshot();
	} );
} );
