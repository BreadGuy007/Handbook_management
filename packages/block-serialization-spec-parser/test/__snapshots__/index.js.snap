// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`block-serialization-spec-parser-js basic parsing parse() works properly 1`] = `
Array [
  Object {
    "attrs": Object {},
    "blockName": "core/more",
    "innerBlocks": Array [],
    "innerContent": Array [
      "<!--more-->",
    ],
    "innerHTML": "<!--more-->",
  },
]
`;

exports[`block-serialization-spec-parser-php basic parsing parse() works properly 1`] = `
Array [
  Object {
    "attrs": Array [],
    "blockName": "core/more",
    "innerBlocks": Array [],
    "innerContent": Array [
      "<!--more-->",
    ],
    "innerHTML": "<!--more-->",
  },
]
`;
